cmd/cloud/server.go

```diff
@@ -127,6 +127,8 @@ var serverCmd = &cobra.Command{
 		keepFilestoreData, _ := command.Flags().GetBool("keep-filestore-data")
 		useExistingResources, _ := command.Flags().GetBool("use-existing-aws-resources")
 
+		model.UtilityValuesDirectory, _ = command.Flags().GetString("utility-values-directory")
```

Comment:
Can we do a directory-exists check here and at least log a warning if it doesn't?

Reply:
Sure sure

Reply:
This should actually be an error case, if the directory doesn't exist, so I'm just going to have the process fail to start if it is not missing, or otherwise there will be weird behavior much later in execution.

The only reason I did not include this check to begin with is that I was thinking it would be more appropriate for the PR when the `helm-charts` directory is actually removed, but if I'm going to add this check, I think it should be a fatal error if the directory is not found.

Reply:
Note that this change essentially deprecates the `helm-charts` directory, but I think it makes sense to deploy this code, see the git-sync sidecar in action, and then remove the `helm-charts` directory as part of the next deployment. Let me know if any of these assumptions are bad ideas.